metrics/cli/test/common/model/metrics_web_config_test.dart

```diff
@@ -0,0 +1,45 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+import 'package:cli/common/model/metrics_web_config.dart';
+import 'package:cli/sentry/model/sentry_config.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group("MetricsWebConfig", () {
+    const clientId = 'clientId';
+    const sentryDsn = 'sentryDsn';
+    const sentryEnvironment = 'sentryEnvironment';
+    const sentryRelease = 'sentryRelease';
+    const sentryConfig = SentryConfig(
+      dsn: sentryDsn,
+      environment: sentryEnvironment,
+      release: sentryRelease,
+    );
+
+    test("creates an instance with the given values", () {
+      final config = MetricsWebConfig(
+        googleSignInClientId: clientId,
+        sentryConfig: sentryConfig,
+      );
+
+      expect(config.googleSignInClientId, equals(clientId));
+      expect(config.sentryConfig, equals(sentryConfig));
+    });
```

Comment:
Would it be great to add a trailing comma here to be consistent with other tests in this project? 
```suggestion
    },);
```